persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/ClinicalDataMapper.java

```diff
@@ -16,13 +16,13 @@
 
     List<ClinicalData> getPatientClinicalData(List<String> studyIds, List<String> patientIds, List<String> attributeIds, 
                                               String projection, Integer limit, Integer offset, String sortBy, 
-                                              String direction);
+                                              String direction, Boolean includeSample);
```

Comment:
can we add sample information at the service layer instead?

Reply:
I do not think performance will be an issue here. Patient and Sample are different entities having its own service it might be benefit in the long run.

Reply:
True, but patient and sample are related. May be I could remove this field and add it as new projection. what do you think?

Reply:
let me think about it and get back to you.

Reply:
@n1zea144 , @sheridancbio , can we please have some inputs on this. Thanks!

Reply:
Yes, we do grab study info.
Yes, a new projection beside `DETAILED` (fetching more information about clinical attribute). This new projection can be hardcoded/internal as it currently is used at limited places(only in related to study-view related database queries)

Reply:
I like your idea about using a projection, but don't like introducing a new one.  You don't want to use  `DETAILED` because it returns extra information?  Is it prohibitive to just throw the extra info away?